src/metpy/calc/indices.py

```diff
@@ -152,23 +152,33 @@ def mean_pressure_weighted(pressure, *args, height=None, bottom=None, depth=None
 @preprocess_and_wrap()
 @check_units('[pressure]', '[speed]', '[speed]', '[length]')
 def bunkers_storm_motion(pressure, u, v, height):
-    r"""Calculate the Bunkers right-mover and left-mover storm motions and sfc-6km mean flow.
+    r"""Calculate right-mover and left-mover supercell storm motions using the Bunkers method.
 
-    Uses the storm motion calculation from [Bunkers2000]_.
+    This is a physically based, shear-relative, and Galilean invariant method for predicting
+    supercell motion. Full atmospheric profiles of wind components, as well as pressure and
+    heights, need to be provided so that calculation can properly calculate the required
+    sfc-6km mean flow.
+
+    The calculation boils down to (from [Bunkers2000]_):
+
+    * 0–6-km non-pressure-weighted mean wind
+    * a deviation from the 0–6-km mean wind of 7.5 m s−1
+    * a 5.5–6-km mean wind for the head of the vertical wind shear vector
+    * a 0–0.5-km mean wind for the tail of the vertical wind shear vector.
```

Comment:
I haven't looked in the rendered docs, but the `<num><dash><num><dash><unit>` representations here are a little tough to read.